### PR TITLE
feat(text-pdf): add pdf-raw route for pdf file (instead of url)

### PR DESCRIPTION
**Pull request by leogail**

### CODE REVIEW

File: services/pdf-text/v1/pdf-raw.ini [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# OpenAPI Documentation - JSON format (dot notation)
+mimeType = application/json
+
+post.operationId = post-v1-pdf-raw
+post.summary = Extrait le texte d'un fichier PDF
+post.description = Transforme un fichier type PDF en texte
+post.tags.0 = pdf-text
+post.requestBody.required = true
+post.responses.default.description = PDF au format PDF
```
Comment (parmentf) [inline services/pdf-text/v1/pdf-raw.ini:9]: **nitpick (non-blocking):** ‏The response is a PDF file?!?

File: services/pdf-text/v1/pdf-raw.ini [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+post.requestBody.content.application/pdf.schema.type = string
+post.requestBody.content.application/pdf.schema.format = binary
+post.responses.default.content.application/pdf.schema.type = string
+post.responses.default.content.application/pdf.schema.format = binary
```
Comment (parmentf) [inline services/pdf-text/v1/pdf-raw.ini:13]: **nitpick (non-blocking):** ‏I thought response was a JSON?

File: services/pdf-text/v1/pdf-raw.py [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+@author: cuxac
+"""
+
+import subprocess
+import re
+import spacy
+import requests
+import os
+import unidecode
+import random
```
Comment (parmentf) [inline services/pdf-text/v1/pdf-raw.py:15]: **suggestion:** ‏Seems to be unused
```suggestion
```

File: services/pdf-text/v1/pdf-raw.py [hunk 4]
```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul  3 11:42:06 2023
+
+@author: cuxac
```
Comment (parmentf) [inline services/pdf-text/v1/pdf-raw.py:6]: **quibble (non-blocking):** ‏Better set the real one or not set anyone
```suggestion
@author: leogail
```

File: services/pdf-text/v1/pdf-raw.py [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+    regex_doi = r"10\.\d{4,9}/[-._;()/:A-Z0-9]+"
+
+    contient_caracteres_speciaux = re.search(regex_caracteres_speciaux, chaine)
+    contient_url = re.search(regex_url, chaine)
+    contient_doi = re.search(regex_doi, chaine)
+
+    return not (contient_caracteres_speciaux or contient_url or contient_doi)
+
+def get_alphabetic_numeric_ratio(chaine):
+    # Initlialize count variables
```
Comment (parmentf) [inline services/pdf-text/v1/pdf-raw.py:100]: **typo:** in comment‏
```suggestion
    # Initialize count variables
```

File: services/pdf-text/v1/pdf-raw.py [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+        for line in lines:#[0:220]:
+            # Extract font information
+        
+            match = font_regex.search(line)
+            if match:
+                current_font = match.group(1)
+    
+            else:
+                current_font = None
+            if current_font!=None:
```
Comment (parmentf) [inline services/pdf-text/v1/pdf-raw.py:163]: **polish:** ‏Linter says...
```suggestion
            if current_font is not None:
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: services/pdf-text/v1/pdf-raw.py (hunk 4 above)
Comment (parmentf) [reply services/pdf-text/v1/pdf-raw.py:6]: If your editor don't set these metadata automatically, remove them (did you create the file on July, 3rd?).

File: services/pdf-text/v1/pdf-raw.ini (hunk 1 above)
Comment (leogail) [inline services/pdf-text/v1/pdf-raw.ini:9]: **praise:** 
```suggestion
post.responses.default.description = références au fomart json
```

File: services/pdf-text/v1/pdf-raw.ini (hunk 2 above)
Comment (leogail) [inline services/pdf-text/v1/pdf-raw.ini:13]: **praise:** 
```suggestion
post.responses.default.content.application/json.schema.$ref
```

**Review by parmentf — DISMISSED** [review]

File: services/pdf-text/v1/pdf-raw.ini [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+post.requestBody.required = true
+post.responses.default.description = références au fomart json
+post.requestBody.content.application/pdf.schema.type = string
+post.requestBody.content.application/pdf.schema.format = binary
+post.responses.default.content.application/json.schema.$ref
+post.parameters.0.description = Indenter le JSON résultant
+post.parameters.0.in = query
+post.parameters.0.name = indent
+post.parameters.0.schema.type = boolean
+
```
[the remark this answers is not in the document]
Comment (leogail) [reply services/pdf-text/v1/pdf-raw.ini:17]: Done

**Review by parmentf — APPROVED** [review]

**Pull request merged**